recipes/octo-logger-cpp/all/conanfile.py

```diff
@@ -52,9 +52,9 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("fmt/10.1.1", transitive_headers=True)
+        self.requires("fmt/10.2.1", transitive_headers=True)
```

Comment:
```suggestion
        self.requires("fmt/10.1.1", transitive_headers=True)
```

Reply:
I'm proposing this change this because we are asking not to bump dependencies unless there's a justification for that, that's because it now introduces version conflict problems, we hope this won't be necessary on the future when we finish working in the new CCI pipeline but we would appreciate if this is avoided unless strictly necessary for the moment. Please [check this comment](https://github.com/conan-io/conan-center-index/pull/23233#issuecomment-2022388759) for more info.

Reply:
@czoido
OK. I understand.
I am sad to see CCI lose the convenience of being able to actively use new libraries, but I agree that it is natural for a service provider to prioritize stables.